akka-stream-tests/src/test/scala/akka/stream/io/TlsSpec.scala

```diff
@@ -113,11 +114,39 @@ class TlsSpec extends StreamSpec(TlsSpec.configOverrides) with WithLogCapturing
       x
     }
 
-    val cipherSuites =
-      NegotiateNewSession.withCipherSuites("TLS_ECDHE_RSA_WITH_AES_128_CBC_SHA", "TLS_RSA_WITH_AES_128_CBC_SHA")
-    def clientTls(closing: TLSClosing) = TLS(sslContext, None, cipherSuites, Client, closing)
-    def badClientTls(closing: TLSClosing) = TLS(initWithTrust("/badtruststore"), None, cipherSuites, Client, closing)
-    def serverTls(closing: TLSClosing) = TLS(sslContext, None, cipherSuites, Server, closing)
+    def createSSLEngine(context: SSLContext, role: TLSRole): SSLEngine =
+      createSSLEngine2(context, role, hostnameVerification = false, hostInfo = None)
+
+    def createSSLEngine2(
```

Comment:
Can it be another overload for `createSSLEngine`? Otherwise, doesn't really matter.

Reply:
yeah, that was weird. compiler said no (or if it was just IntelliJ?) and I obeyed without spending time on it since just test